app/src/main/java/com/greenaddress/abcore/LogActivity.java

```diff
@@ -89,4 +99,41 @@ protected void onResume() {
             }
         }
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(final Menu menu) {
+        final MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(final MenuItem item) {
```

Comment:
We should probably extract this `onOptionsItemSelected` method to some utility class, instead of duplicating it

Reply:
Removing the top menu (as mentioned in the other comment) from all non Dashboard activities will solve this problem.

Reply:
Fixed with the latest commit. The menu code is not repeated because its only in the Main activity.